### PR TITLE
add support for Dpcloudserver

### PR DESCRIPTION
**Pull request by felix5572**
add support for Dp cloud server

### CODE REVIEW

File: dpdispatcher/dpcloudserver/config.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+HTTP_TIME_OUT = 30
+API_HOST = "http://cloudserver.deepmd.net"
+
+
+ALI_STS_ENDPOINT = 'http://oss-cn-shenzhen.aliyuncs.com'
+ALI_STS_BUCKET_NAME = 'dpcloudserver'
```
Comment (xiaoyeqiannian) [inline dpdispatcher/dpcloudserver/config.py:7]: create dispatcher config all in one is better

File: dpdispatcher/dp_cloud_server_context.py [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+from .dpcloudserver import api
+from .dpcloudserver import zip_file
+# from zip_file import zip_files
+DP_CLOUD_SERVER_HOME_DIR = os.path.join(
+    os.path.expanduser('~'),
+    '.dpdispatcher/',
+    'dp_cloud_server/'
+)
+ENDPOINT = 'http://oss-cn-shenzhen.aliyuncs.com'
+BUCKET_NAME = 'dpcloudserver'
```
Comment (xiaoyeqiannian) [inline dpdispatcher/dp_cloud_server_context.py:17]: need save in config file

File: dpdispatcher/dp_cloud_server.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+        job.job_id = job_id
+        job.job_state = JobStatus.waiting
+        return job_id
+
+    def check_status(self, job):
+        if job.job_id == '':
+            return JobStatus.unsubmitted
+        print('debug: check_status', job)
+        dp_job_status = api.get_tasks(job.job_id)[0]["status"]
```
[the remark this answers is not in the document]
Comment (felix5572) [reply dpdispatcher/dp_cloud_server.py:79]: ok, good idea!

File: dpdispatcher/dpcloudserver/config.py (hunk 1 above)
Comment (felix5572) [reply dpdispatcher/dpcloudserver/config.py:7]: OK,

**Pull request merged**